.tekton/tasks/ec-checks.yaml

```diff
@@ -11,19 +11,33 @@ metadata:
 spec:
   description: >-
     This task can be used to run enterprise contract checks
+  stepTemplate:
+    securityContext:
+      # runAsUser is needed because the different images used in this Task run as different users. Ideally,
+      # we'd let OpenShift do its thing and set a random UID with a common group ID for *all* containers
+      # in the namespace, but that is currently disabled so it's wild west.
+      runAsUser: 0
+    env:
+      - name: HOME
```

Comment:
This is needed so EC finds the authfile :shrug: 
And it also, for some reason, makes `runAsUser` required. Quite odd.